### PR TITLE
feat(tables): adapt for mobiles

### PR DESCRIPTION
**Pull request by andreybakanovsky**

### CODE REVIEW

**Comment by andreybakanovsky** [comment]
@VladislavSokov could you please check public share conditionals

File: app/views/accounts/show.html.slim [hunk 1]
```diff
@@ -26,66 +27,69 @@
...
-      - unless public_share?(account)
-        .card-footer
-          = link_to '+ Add', new_account_account_automatic_topup_config_path(account), class: 'card-footer-item'
+        .card-header-title.is-centered
+          .title.is-4 Weekly Automatic Topups
+      .card-content.px-1
+        div.table-container
+          table.table.is-fullwidth.is-striped
+            thead
+              tr
```
Comment (VladislavSokov) [inline app/views/accounts/show.html.slim:76]: Perhaps we can add the `.has-text-centered` class?
What do you think about this?

**Comment by VladislavSokov** [comment]
> @VladislavSokov could you please check public share conditionals

Llooks correct.

File: app/views/spends/new.html.slim [hunk 2]
```diff
@@ -14,4 +14,4 @@
         = f.text_field :description, class: "input", placeholder: "Description"
       div.buttons.is-flex.is-justify-content-flex-end
         = link_to 'Back', request.referer, class: 'button is-light'
-        = f.submit 'Save', class: "button is-primary is-fullwidth"
\ No newline at end of file
+        = f.submit 'Save', class: "button is-primary is-fullwidth"
```
[the remark this answers is not in the document]
Comment (andreybakanovsky) [reply app/views/spends/new.html.slim:17]: Added, all right

File: app/views/accounts/show.html.slim [hunk 3]
```diff
@@ -26,66 +27,69 @@
...
-              th From
-              th To
-              th Amount
-              th Description
-              th Actions
-          = render partial: 'transaction', collection: account.transactions, account: account
-
+          h3.title.is-4 History
+      .card-content.px-1
+        div.table-container
```
[the remark this answers is not in the document]
Comment (andreybakanovsky) [reply app/views/accounts/show.html.slim:32]: It possible, thx!

File: app/views/accounts/show.html.slim (hunk 1 above)
Comment (andreybakanovsky) [reply app/views/accounts/show.html.slim:75]: Ok, picked it up, but not implemented.

**Comment by andreybakanovsky** [comment]
@VladislavSokov, thank you!

**Review by VladislavSokov — APPROVED** [review]

**Review by ka8725 — APPROVED** [review]

**Pull request merged**